hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BlockReceiver.java

```diff
@@ -839,7 +840,7 @@ duration, datanodeSlowLogThresholdMs, getVolumeBaseUri(),
           
           replicaInfo.setLastChecksumAndDataLen(offsetInBlock, lastCrc);
 
-          datanode.metrics.incrBytesWritten(len);
+          datanode.metrics.incrBytesWritten(bytesWritten);
```

Comment:
Why not just use 'numBytesToDisk' var?

Reply:
@huangzhaobo99 Yes, sir. We can use bytesWritten here, thanks for you reviewing.